### PR TITLE
Add Modding Mentorship: Cycle 2 Signups newspost

### PR DESCRIPTION
**Pull request by venix12**
we have no banner, not sure if we can go live without one 🤔 
deadlines need to be readjusted depending on when the post goes live

### CODE REVIEW

**Comment by zeusminus** [comment]
> we have no banner, not sure if we can go live without one 🤔 
why don't you use the one used for the previous mentorship newsposts?

**Comment by venix12** [comment]
this is CMP banner, modding mentorship is a different thing so don't think that works well

**Comment by Ephemeralis** [comment]
your options are:

- cajole a designer into making a newsbanner that someone from news-approval (me or peppy) signs off on
- use the generic media banner (same one used in contest page headers for the MBCs, etc)

**Comment by venix12** [comment]
hmm, see no harm in going with the generic banner i guess

File: news/2021-02-04-modding-mentorship-cycle-2-signups-now-open [hunk 1]
```diff
@@ -0,0 +1,25 @@
+---
+layout: post
+title: Modding Mentorship Cycle 2: Signups Now Open
+date: 2021-02-04 10:00:00 +0000
+---
+
+Greetings, modders! With the new year starting, we want to announce the second run of Modding Mentorship! If you ever wanted to polish your modding abilities or assist newer members of modding communities in their way to achieve the needed modding skills, this is your time to shine!
+
+The Modding Mentorship is a place where new and promising modders of each gamemode can improve their modding skills under the supervision of pre-selected pool of modding experts serving as mentors.
```
Comment (zeusminus) [inline news/2021-02-04-modding-mentorship-cycle-2-signups-now-open:9]:
```suggestion
The Modding Mentorship is a place where new and promising modders of each gamemode can improve their modding skills under the supervision of a pre-selected pool of modding experts serving as mentors.
```

File: news/2021-02-04-modding-mentorship-cycle-2-signups-now-open [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+During the cycle, you will get to know the basics of modding as well as more in-depth analysis from the mentors. You will also have an opportunity to participate in a variety of events (like modding puzzles).
+
+If you believe you've got what it takes to teach a new generation of modders, don't hesitate, and try yourself as a mentor! [The information thread](https://osu.ppy.sh/community/forums/topics/1218245) will guide you on how to apply right away! If you are just taking your first steps in the modding community, though, consider applying as a mentee!
```
Comment (zeusminus) [inline news/2021-02-04-modding-mentorship-cycle-2-signups-now-open:13]:
```suggestion
If you believe you've got what it takes to teach a new generation of modders, don't hesitate and try yourself as a mentor! [The information thread](https://osu.ppy.sh/community/forums/topics/1218245) will guide you on how to apply right away! If you are just taking your first steps in the modding community, though, consider applying as a mentee!
```

File: news/2021-02-04-modding-mentorship-cycle-2-signups-now-open [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+You may also want to join us at [the Modding Mentorship discord](https://discord.gg/TyvrMvR) to participate in the public events and keep up with the announcements.
+
+---
+
+For this cycle, the application deadlines will go as following:
+- Mentor signups will be open until **18th February, 2021**
+- Mentee signups will open on **18th February, 2021** and close on **25th February**
```
Comment (zeusminus) [inline news/2021-02-04-modding-mentorship-cycle-2-signups-now-open:21]:
```suggestion
For this cycle, the application deadlines will go as follows:
- Mentor signups will be open until **February 18, 2021**
- Mentee signups will open on **February 18, 2021** and close on **February 25**
```
these are the correct date & time formatting according to the [Article Styling Criteria](https://osu.ppy.sh/wiki/en/Article_styling_criteria/Writing#date-and-time)

File: news/2021-02-04-modding-mentorship-cycle-2-signups-now-open [hunk 4]
```diff
@@ -0,0 +1,25 @@
+---
+layout: post
+title: Modding Mentorship Cycle 2: Signups Now Open
+date: 2021-02-04 10:00:00 +0000
+---
+
+Greetings, modders! With the new year starting, we want to announce the second run of Modding Mentorship! If you ever wanted to polish your modding abilities or assist newer members of modding communities in their way to achieve the needed modding skills, this is your time to shine!
```
Comment (zeusminus) [inline news/2021-02-04-modding-mentorship-cycle-2-signups-now-open:7]:
```suggestion
Greetings, modders! With the new year starting, we want to announce the second run of Modding Mentorship! If you ever wanted to polish your modding abilities or assist novice modders in their way to achieving the needed modding skills, this is your time to shine!
```
the sentence sounds a little weird when read carefully, you might want to simplify it the way I suggested

**Review by zeusminus — CHANGES_REQUESTED** [review]
There isn't much to suggest, nonetheless skimmed through the article as best as I can. However, you seem to have forgotten to add `.md` extension after the filename, thus the contents were parsed as plain text.

**Comment by Ephemeralis** [comment]
this sort of fell into the ether, will it still be relevant after a redate? @venix12

**Comment by venix12** [comment]
yes

**Comment by Ephemeralis** [comment]
rejuggle the dates around to whatever fits and ping/dm me on Discord and we can slot this in on any date that isn't Wed/Sat

**Comment by TicClick** [comment]
today isn't Wed/Sat

**Pull request closed without merging**